.github/dependabot.yml

```diff
@@ -37,3 +40,13 @@ updates:
     labels:
       - "theme/dependencies"
       - "theme/website"
+  - package-ecosystem: github-actions
+    open-pull-requests-limit: 5
+    directory: /
+    labels:
+      - "theme/dependencies"
+      - "theme/ci"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+      time: "09:00"
```

Comment:
@dduzgun-security you'll want to fix your editor to make sure files have their EOF on save. Not having this ends up with surprising behavior for various Unix tools:

```
wc -l .github/dependabot.yml
51 .github/dependabot.yml
```